crates/glaredb/src/bin/main.rs

```diff
@@ -74,10 +72,18 @@ enum Commands {
         #[clap(short, long, value_parser)]
         password: Option<String>,
 
-        /// Optional file path to store metastore data (to enable persistent
-        /// data storage when in-process store is launched).
+        /// Optional file path for persisting data.
+        ///
+        /// Catalog data and user data will be stored in this directory.
         #[clap(short = 'f', long, value_parser)]
-        local_file_path: Option<PathBuf>,
+        data_dir: Option<PathBuf>,
+
+        /// Path to GCP service account to use when connecting to GCS.
+        ///
+        /// Sessions must be proxied through pgsrv, otherwise attempting to
+        /// create a session will fail.
+        #[clap(short, long, value_parser)]
+        service_account_path: Option<String>,
```

Comment:
@f0ssel We need to supply this when running `glaredb server ...` in terraform and for dedicated deployments.